flipperdevices/flipperzero-firmware
# NFC: Mf Desfire fix reading big files

# What's new

- Rework reading long data and record sizes
- Limit file data rendering to 256 bytes per record

# Verification 

- Create big file:
`hf mfdes formatpicc` - format desfire
`hf mfdes createapp --aid 001122` - create application
`hf mfdes createfile --aid 001122 --fid 01 --amode plain --rrights free --wrights free --rwrights free --chrights free --size 001d01` - create big file
- Read file with flipper. Verify that only 256 bytes is visible
- Save file. Verify that all data is saved on sd card.

# Checklist (For Reviewer)
